PluginCore/PluginCore/Resources/en_US.resX

```diff
@@ -6018,4 +6018,16 @@ Custom locales must be an extension of a default locale, e.g. en-US.</value>
     <value>Keeps the caret centered when caret is navigated to.</value>
     <comment>Added after 5.1.1</comment>
   </data>
+  <data name="CodeRefactor.Description.ReplaceAllOccurrences" xml:space="preserve">
+    <value>Replace all highlighted occurrences of initial expression with new variable.</value>
+    <comment>Added after 5.2.0</comment>
+  </data>
+  <data name="CodeRefactor.Description.ReplaceInitialOccurrence" xml:space="preserve">
+    <value>Replace initial expression only.</value>
```

Comment:
What about `Only this occurence / Select all 3 occurences` (should appear only if there is more than one occurrence.